test/testfloat/testfloat.cpp

```diff
@@ -0,0 +1,395 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <fizzy/execute.hpp>
+#include <fizzy/parser.hpp>
+#include <test/utils/floating_point_utils.hpp>
+#include <test/utils/hex.hpp>
+#include <cfenv>
+#include <iomanip>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <unordered_map>
+
+using namespace fizzy::test;
+
+namespace
+{
+constexpr auto USAGE = R"(Fizzy TestFloat
+
+Tool for testing IEEE 754 floating-point compliance.
+Expects inputs from TestFloat's testfloat_gen.
+See http://www.jhauser.us/arithmetic/TestFloat-3/doc/TestFloat-general.html.
+
+Usage:
+  fizzy-testfloat [options] <function>
+
+  -rnear_even     Round to nearest/even [default].
+  -rminMag        Round to minimum magnitude (toward zero).
```

Comment:
I think these options are inconsistent, underscore vs camelcase.

I'd suggest: `-rnearest` and `-rmin-mag`.

Reply:
These follow options from `testfloat_gen` so you don't need to remember two names doing the same thing. This usually goes like this
```
testfloat_gen -rminMag f32_add | fizzy-testfloat -rminMag f32_add
```

See https://github.com/wasmx/fizzy/pull/511#issue-473460885.

Reply:
Can you leave a note in this source file?